website/vercel.json

```diff
@@ -870,114 +870,6 @@
       "source": "/docs/plugins/destinations/sqlite/:path*",
       "destination": "https://hub.cloudquery.io/plugins/destination/cloudquery/sqlite"
     },
-    {
```

Comment:
We have to remove the existing redirects otherwise https://github.com/cloudquery/cloudquery/pull/16681/files#diff-21378a71578c160a16aa86a2ccc6120c09d1d10a3deaf16af651a228bd0cfde4R1054 won't match